src/main/java/io/singularitynet/MissionHandlers/ObservationFromTriplesImplementation.java

```diff
@@ -0,0 +1,120 @@
+package io.singularitynet.MissionHandlers;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonPrimitive;
+import io.singularitynet.MissionHandlerInterfaces.ICommandHandler;
+import io.singularitynet.MissionHandlerInterfaces.IObservationProducer;
+import io.singularitynet.projectmalmo.MissionInit;
+import io.singularitynet.projectmalmo.ObservationFromTriple;
+import net.minecraft.block.Block;
+import net.minecraft.client.MinecraftClient;
+import net.minecraft.item.Item;
+import net.minecraft.registry.Registry;
+import net.minecraft.registry.tag.TagKey;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
+import static net.minecraft.registry.Registries.ITEM;
+
+public class ObservationFromTriplesImplementation extends HandlerBase implements IObservationProducer, ICommandHandler {
+    private boolean sendRec;
+
+    @Override
+    public boolean isOverriding() {
+        return false;
+    }
+
+    @Override
+    public void setOverriding(boolean b) {
+
+    }
+
+    @Override
+    public void install(MissionInit currentMissionInit) {
+        sendRec = false;
+    }
+
+    @Override
+    public void deinstall(MissionInit currentMissionInit) {
+
+    }
+
+    @Override
+    public boolean execute(String command, MissionInit currentMissionInit) {
+        String comm[] = command.split(" ", 2);
+        if (comm.length == 2 && comm[0].equalsIgnoreCase(ObservationFromTriple.TRIPLES.value())){
+            if (comm[1].equalsIgnoreCase("off")) {
+                this.sendRec = false;
+            } else {
+                this.sendRec = true;
+            }
+            return true;
+        }
+        return false;
+    }
+
+    @Override
+    public void cleanup() {
+
+    }
+
+    @Override
+    public void prepare(MissionInit missionInit) {
+
+    }
+
+    @Override
+    public void writeObservationsToJSON(JsonObject json, MissionInit currentMissionInit) {
+        if (!this.sendRec){
+            return;
+        }
+        Registry<Item> str_ent =  MinecraftClient.getInstance().world.getRegistryManager().get(ITEM.getKey());
+        List<Item> list_ent = str_ent.stream().toList();
+        JsonArray triple_array = new JsonArray();
+        Map<Block, Item> from_block = list_ent.get(0).BLOCK_ITEMS;
```

Comment:
What is in list_ent? Does it have more items or it is list of lenght 1?

Reply:
> What is in list_ent? Does it have more items or it is list of lenght 1?

List of all items. But each item contains this Map<Block, Item> entirely.